src/pages/projectSettings/Tabs.tsx

```diff
@@ -45,6 +45,10 @@ export const ProjectSettingsTabs: React.FC<Props> = ({
   const { setInitialData } = useProjectSettingsContext();
 
   const projectId = projectData?.projectRef?.id;
+  const repoId = repoData?.projectRef?.id;
+  const {
+    projectRef: { identifier },
+  } = projectData ?? { projectRef: {} };
```

Comment:
Is there a reason this can't be `const identifier = projectData?.projectRef?.identifier`?